packages/fstar/fstar.0.9.5.0/opam

```diff
@@ -33,6 +34,6 @@ remove: [
       "%{prefix}%/bin/fstar"
       "%{prefix}%/share/fstar" ]
 ]
-available: [ ocaml-version >= "4.02.3" & (ocaml-version < "4.03.0" | ocaml-version >= "4.04.0") ]
+available: [ ocaml-version >= "4.02.3" & (ocaml-version < "4.03.0" | (ocaml-version >= "4.04.0" & ocaml-version < "4.07.0")) ]
```

Comment:
I'm a bit confused by this expression, both before and after the change. What is it trying to say?

Reply:
In fact, before and after the change, this expression says that we do not support 4.03.x.
After the change: we do not support 4.07.x onwards (in fact, I tried to test the opam package with the 4.07.0+trunk switch, but opam refuses to proceed due to version conflicts.)
Anyway, regardless of this issue, we need to explicitly put an upper bound on the supported version of OCaml, because every time OCaml ships a new version, we need to manually add a corresponding `FStar_Extraction_ML_PrintML_y.yy.X.ml`